students/JaeKim/lesson07/assignment/test_database.py

```diff
@@ -0,0 +1,42 @@
+import database as d
+import pytest
+
```

Comment:
I don't understand what this file is testing. There is no database.py in this directory. Where is it getting database?